src/plugins/configuration_provider/server/routes/index.ts

```diff
@@ -0,0 +1,163 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { schema } from '@osd/config-schema';
+import {
+  IRouter,
+  IScopedClusterClient,
+  Logger,
+  OpenSearchDashboardsRequest,
+  OpenSearchDashboardsResponseFactory,
+} from '../../../../core/server';
+import { ConfigurationClient } from '../types';
+
+export function defineRoutes(
+  router: IRouter,
+  getConfigurationClient: (inputOpenSearchClient: IScopedClusterClient) => ConfigurationClient,
+  logger: Logger
+) {
+  router.get(
+    {
+      path: '/api/configuration/existsCspRules',
```

Comment:
i think get `api/configuration/csp` or `api/config/csp` would be ok and then use `get` `post` put`

Reply:
Sure. Updated the path structure.